src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -100,11 +105,28 @@ final class CachedAppConfiguration {
 					// server response HTTP 304 is considered a 'successful fetch'
 					self.store.lastAppConfigFetch = Date()
 				default:
-					// ensure reset
-					self.store.lastAppConfigETag = nil
-					self.store.lastAppConfigFetch = nil
+					// try to provide the default configuration or return error response
+					guard
+						let url = self.defaultAppConfigPath,
+						let data = try? Data(contentsOf: url),
+						let zip = Archive(data: data, accessMode: .read),
+						let defaultConfig = try? zip.extractAppConfiguration()
+					else {
+						assertionFailure("Should not happen. Check config deserialization!")
+						Log.error("Providing default app configuration failed! Initial HTTP error: \(error)")
+						self.store.lastAppConfigETag = nil
+						self.store.lastAppConfigFetch = nil
+						self.completeOnMain(completion: completion, result: .failure(error))
```

Comment:
I would love to see a `fatalerror` here:
- If the file is not contained in the binary something went really wrong
- This would allow us to loose the `result` type and just have a completion that always contains an appconfig. I think this would make the whole feature really take off 🚀